devel/ansible/roles/hotness-dev/files/fedmsg.d/hotness.py

```diff
@@ -48,8 +47,14 @@
     'hotness.koji': {
         'server': 'https://koji.fedoraproject.org/kojihub',
         'weburl': 'http://koji.fedoraproject.org/koji',
-        'cert': os.path.expanduser('~/.fedora.cert'),
-        'ca_cert': os.path.expanduser('~/.fedora-server-ca.cert'),
+        # Kerberos configuration to authenticate with Koji. In development
+        # environments, use `kinit <fas-name>@FEDORAPROJECT.ORG` to get a
+        # Kerberos ticket and set all krb_* settings to `None`.
+        'krb_principal': None,
+        'krb_keytab': None,
+        'krb_ccache': None,
+        'krb_proxyuser': None,
```

Comment:
I still think this shouldn't be configurable, or at the very least not in the default config, since people might start asking to use it.